content/post/roadmap.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Creating a Data roadmap"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["strategy", "management"]
+date: 2018-12-22T13:19:40-05:00
+draft: true
+---
+
+As the new year rolls around, many Data leaders are thinking about (or have already created) 2019 road maps for their team and function. Since Data often works cross functionally with other teams, it's key that you consider other team's priorities and objectives in developing your road map. Below is a blueprint you can use to get started.
+
+<!--more-->
+
+Your road map is a tool to:
+<li> Force you to make explicit what your team will be working on, e.g., "Improve existing data infrastructure" is not specific enough or helpful. "Implement a [star schema](https://en.wikipedia.org/wiki/Star_schema) and migrate business transformations from pure SQL to [dbt](https://www.getdbt.com)" is better.
+<li> Communicate your team's priorities to leaders across the organization, e.g., When your team is focused on data infrastructure for all of Q1, stakeholders may be left unsatisfied. It's helpful to identify how each of the various Data infrastructure projects (i.e., adding testing and error alerting, data re-model, `dbt`) will ultimately support stakeholder initiatives slated for Q2 and beyond.
+<li> Clarify (and hopefully excite) your team for the year ahead. The road map is an opportunity to be ambitious and set up your team to think about what their year can conceivably look like.
+<li> Help defend head count for future hiring needs and also focus your hiring search, e.g., When Finance asks you why you need 7 more head count through Q3, a road map will allow you to identify what the break down is (junior vs. senior, analyst vs. data scientist) and to what projects they would be assigned. 
+ 
+### Identify audience for such a document
+
+Ideally everyone in the organization has access to this road map. My preferred medium for communicating my road map is presentation. I can inject visuals if needed, color code where appropriate and use the natural aspect of "one topic per slide" to help navigate. 
+
+While everyone should have access, you don't need to make it digestible to everyone in the organization. You key audience should be leadership, relevant cross-functional partners and your team. That will help guide the level of detail required. I also like to present my road map to stakeholders, which allows me to keep it high level in the presentation and speak to it when necessary.
```

Comment:
> I also like to present my road map to stakeholders, which allows me to keep it high level in the presentation and speak to it when necessary.

I'm not sure I follow what the 'it' in this sentence is ... is it the roadmap? or specific concerns?

Reply:
Agreed. Tried to add clarity.